### PR TITLE
Swift 2.3

### PR DESCRIPTION
**Pull request by danielaRiesgo**
## Summary ##

Migration to swift 2.3 and ios scripts updating.

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,6 +1,3 @@
 # Core & Functional Programming
-github "ReactiveCocoa/ReactiveCocoa" == 4.2.0
-github "neilpa/Rex" ~> 0.10.0
-
-# Utilities
-github "CocoaLumberjack/CocoaLumberjack" ~> 2.2.0
+github "ReactiveCocoa/ReactiveCocoa" ~> 4.2.2
+github "RACCommunity/Rex" "master"
```
Comment (guidomb) [inline Cartfile]: Do we need to depend on `master`? Is it possible to use a specific release?

**Comment by pablog** [comment]
@danielaRiesgo don't forget to remove cocoa lumberjack from "Carthage" build phase.

**Comment by danielaRiesgo** [comment]
@pablog yes, already did and squashed when rebased with "Eliminated CocoaLumberjack from project."

**Comment by pablog** [comment]
Great!

**Pull request merged**